### ISSUE TITLE
Confirm 3 known instances to validate patterns

### ISSUE THREAD

**Issue by spier** [InnerSourceCommons/InnerSourcePatterns#186]
To start using the [new pattern maturity levels](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/contributor-handbook.md#maturity-levels) outlined as part of the new contribution process, we want to move all existing patterns to a folder structure, indicating which maturity each pattern is in.

First we do this:
https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/185

Once that is done, we want to identify the patterns that are used by at least 3 companies (i.e. at least 3 known instances).
These patterns will then be moved from [2-structured][] to [3-validated][].

We will split up the existing patterns between people in our ISC Patterns community that are interested in doing that work. @lenucksi @maxcapraro and @spier have already volunteered for this. But we would love help from anybody really :)

[2-structured]: https://github.com/InnerSourceCommons/InnerSourcePatterns/tree/master/patterns/2-structured
[3-validated]: https://github.com/InnerSourceCommons/InnerSourcePatterns/tree/master/patterns/3-validated

**Comment by fwan2000** [comment]
Here's an idea for your consideration to find the instances for the patterns. How about showing the mindmap of patterns to the general slack channel and soliciting input there? Maybe they can follow the link in mindmap to the pattern md and open a PR to add their instance, or link to a talk that mentioned the pattern?

When would be a good time to do this? After all the patterns regrouping merged to main? Then separate PR to move each pattern up?

**Comment by spier** [comment]
First pattern with sufficient amount of Known Instances seems to be:
https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/innersource-portal.md

**Comment by spier** [comment]
This one is also close (already has 2 Known Instances):
https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/trusted-committer.md

**Comment by spier** [comment]
I scripted this now, so I believe at the point of writing, these are the patterns that have at least 3 known instances:

```
## 2-Structured patterns primed for upgrade to 3-Validated (based on Known Instances only)
3 | ../../patterns/2-structured/30-day-warranty.md
5 | ../../patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md
3 | ../../patterns/2-structured/document-your-guiding-principles.md
3 | ../../patterns/2-structured/trusted-committer.md
3 | ../../patterns/2-structured/core-team.md
3 | ../../patterns/2-structured/maturity-model.md
5 | ../../patterns/2-structured/innersource-portal.md
3 | ../../patterns/2-structured/gig-marketplace.md
3 | ../../patterns/2-structured/project-setup/base-documentation.md
3 | ../../patterns/2-structured/project-setup/communication-tooling.md
```

**Comment by spier** [comment]
#772 is replacing this issue here.
